File: challenges/4-fizz-buzz/javascript/codinghobbit/desafio4.js
function fizzBuzz(x, start = 1) {
  start % 3 == 0 && start % 5 == 0
    ? console.log("FizzBuzz")
    : start % 3 == 0
    ? console.log("Fizz")
    : start % 5 == 0
    ? console.log("Buzz")
    : console.log(start);

  if (start == x) return;
  return fizzBuzz(x, start + 1);
}
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Insira um inteiro: ", (x) => {
  fizzBuzz(x);
  readline.close();
});

File: challenges/4-fizz-buzz/python/camiscf/desafio4.py
# divisivel por 3 -> fizz
# multiplo de 5 -> buzz
# divisivel de 3 e 5 -> fizzbuzz

def fizzbuzz(n):
    if n % 3 == 0 and n % 5 == 0:
        return 'Fizzbuzz'
    elif n % 3 == 0:
        return 'Fizz'
    elif n % 5 == 0:
        return 'Buzz'
    else:
        return n

if __name__ == '__main__':
    n = int(input())
    for i in range(1, n+1):
        print(fizzbuzz(i))
